bugwarrior/services/__init__.py

```diff
@@ -91,12 +69,17 @@ def __init__(self, main_config, main_section, target):
                 if option:
                     self.add_tags.append(option)
 
-        self.default_priority = 'M'
-        if 'default_priority' in self.config:
-            self.default_priority = self.config.get('default_priority')
-
         log.info("Working on [%s]", self.target)
 
+
+    def _get_config_or_default(self, key, default, as_type = lambda x: x):
```

Comment:
FYI, python convention is to have no spaces around the equals sign in keyword arguments. See the fourth bullet at https://www.python.org/dev/peps/pep-0008/#other-recommendations.